recipes/prophyle/meta.yaml

```diff
@@ -10,7 +10,7 @@ package:
   #sha256:
 
 build:
-  number: 0
+  number: 1
   skip: True # [py27]
   script: pip install --upgrade git+https://github.com/karel-brinda/prophyle
```

Comment:
This one is even better. For your information, you can also use github with a URL similar to this:https://github.com/karel-brinda/prophyle/archive/3a4e3ab0be3e10ad5f15b69bce0749cb812c360d.tar.gz The hash is your latest commit.